### PR TITLE
Feature/seab 816/enduser tests

### PR DESCRIPTION
**Pull request by emlys**

### CODE REVIEW

File: cypress/integration/manualTests/basic-enduser.ts [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+  organizations.forEach(t => {
+    it('organization page and collections work', () => {
+      cy.contains('[data-cy=orgName]', t[0]).click();
+      cy.get('[data-cy=collectionName').should('have.length.of.at.least', 1);
+      cy.get('[data-cy=collectionName')
+        .first()
+        .click();
+      cy.url().should('contain', 'collections');
+      cy.get('[data-cy=collectionEntry]').should('have.length.of.at.least', 1);
```
[the remark this answers is not in the document]
Comment (NatalieEO) [reply cypress/integration/manualTests/basic-enduser.ts:165]: They get ordered by number of stars, but I think it'd make sense to look for Broad specifically.

File: cypress/integration/manualTests/basic-enduser.ts [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  it('versions tab works', () => {
+    goToTab('Versions');
+    cy.url().should('contain', '?tab=versions');
+
+    // check that clicking on a different version goes to that version's url
+    cy.contains('[data-cy=versionName]', version2).click();
+    cy.url().should('contain', url + ':' + version2);
+  });
+
+  it('files tab works', () => {
```
Comment (NatalieEO) [inline cypress/integration/manualTests/basic-enduser.ts:34]: Can you check that the two sub tabs can be clicked and if there is content within them?

File: cypress/integration/manualTests/basic-enduser.ts [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    cy.get('[data-cy=workflowColumn]')
+      .first()
+      .within(() => {
+        cy.get('a').click(); // click on the link to the first workflow
+      });
+  });
+});
+
+// pairs of [workflow URL without version number, verified version number, another verified version number, workflow.trsUrl]
+const workflowVersionTuples = [
```
[the remark this answers is not in the document]
Comment (emlys) [reply cypress/integration/manualTests/basic-enduser.ts:142]: added a CWL and a NFL

File: cypress/integration/manualTests/basic-enduser.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (emlys) [reply cypress/integration/manualTests/basic-enduser.ts:182]: replaced seven bridges with broad

**Pull request merged**